osc/commandline.py

```diff
@@ -7892,7 +7746,8 @@ def do_my(self, subcmd, opts, *args):
     @cmdln.alias('se')
     @cmdln.alias('bse')
     def do_search(self, subcmd, opts, *args):
-        """${cmd_name}: Search for a project and/or package.
+        """
+        Search for a project and/or package
```

Comment:
Maybe commit docstings separately to keep diff small, as they don't seem related.

Reply:
I understand that it's difficult to review, but the change is massive anyway.
I'll probably keep it as it is.